gateway/1.5/modules/ROOT/pages/policies-included-route-weight.adoc

```diff
@@ -0,0 +1,143 @@
+= Traffic Management Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route weight, traffic management
+:page-aliases: policies::policies-included-tls.adoc
```

Comment:
Is this policy replacing the standard TLS policy?

Reply:
Nah, I just started by copying/pasting the tls policy.